react-frontend/src/components/StyleComponents/bannerWithImage.js

```diff
@@ -83,16 +68,17 @@ export const BannerTitle = styled.h1.attrs({
 export const BannerWithImageWrapper = styled.div.attrs({
   className: 'banner',
 })`
-  height: 600px;
-  margin-bottom: 120px;
-  overflow: hidden;
-  position: relative;
+  background-image: url(${(props) => props.backgroundImage});
```

Comment:
This was the one I was looking at and wondering if ` url(${ props.backgroundImage});` would actually work?

Reply:
I tried — it doesn't seem to work:

![image](https://user-images.githubusercontent.com/16946297/96770279-1a771a00-1395-11eb-8c8e-265b1c1c4130.png)